src/main/java/org/jenkinsci/test/acceptance/controller/LocalController.java

```diff
@@ -285,15 +285,15 @@ public void diagnose(Throwable cause) {
     public void tearDown(){
         try {
             if (jenkinsHome.exists()) {
-                FileUtils.forceDelete(jenkinsHome);
+                FileUtils.deleteDirectory(jenkinsHome);
             }
         } catch (IOException e) {
             LOGGER.log(Level.WARNING, "Cleaning up temporary JENKINS_HOME failed, retrying in 5 sec.", e);
             //maybe process is shutting down, wait for a sec then try again
             try {
                 Thread.sleep(5000);
                 if (jenkinsHome.exists()) {
-                    FileUtils.forceDelete(jenkinsHome);
+                    FileUtils.deleteDirectory(jenkinsHome);
```

Comment:
Why was this needed?

Reply:
Oh! I got a stacktrace there and I made a check. I thought I had reverted it, so doing it now.